docs/go/how-to-test-workflow-definitions-in-go.md

```diff
@@ -249,6 +249,13 @@ func (s *UnitTestSuite) Test_ProgressWorkflow() {
 }
 ```
 
+:::note
+
+`RegisterDelayedCallback` can also be used to send [Signals](/docs/concepts/what-is-a-signal).
+When using "signal-with-start", set the delay to `0`.
```

Comment:
Did you want to capitalize > "Signal-with-Start" to match the topic title in the sidebar?